spec/models/user_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) { FactoryGirl.create(:user) }
+
+  it 'is valid with a name, email, and password' do
+    expect(user_1).to be_valid
```

Comment:
what does it test? You already saved it to the database so that means that user is valid